source/sessions/snapshot-sessions.rst

```diff
@@ -185,20 +185,20 @@ distinct response object it sends to the driver.
         atClusterTime : <BsonTimestamp>
     }
 
-The ``atClusterTime`` MUST be stored in the ``ClientSession`` to later be passed as the
-``atClusterTime`` field of the ``readConcern`` with ``snapshot`` level field  in subsequent read operations.
+The ``atClusterTime`` timestamp MUST be stored in the ``ClientSession`` to later be passed as the
+``atClusterTime`` field of a ``readConcern`` with a ``snapshot`` level in subsequent read operations.
```

Comment:
I actually think `the readConcern` reads better than `a readConcern` since the readConcern is not optional for these operations

Reply:
```suggestion
``atClusterTime`` field of the ``readConcern`` with a ``snapshot`` level in subsequent read operations.
```